testutil/test_helper.go

```diff
@@ -292,6 +295,15 @@ func (h *TestHelper) GetMulticlusterNamespace() string {
 	return fmt.Sprintf("%s-multicluster", h.GetLinkerdNamespace())
 }
 
+// GetMulticlusterContexts returns a map with the context names for the clusters
+// used in the test
+func (h *TestHelper) GetMulticlusterContexts() map[string]string {
+	return map[string]string{
+		"src": h.multiclusterSrcCtx,
+		"tgt": h.multiclusterTgtCtx,
```

Comment:
big nit: maybe just call these key values `source` and `target`?

Reply:
Yeah good point. I've changed it and introduced some variables to make it easier to change the key values in the future.